packages/fast-data-utilities-react/src/__tests__/components/children.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+
+export interface ObjectWithChildren {
+    nestedObjectChildren: JSX.Element;
+}
+
+export interface ChildrenProps {
+    /**
+     * Children which are restricted to a subset of childOptions via the JSON schema
```

Comment:
what are "childOptions" in this context? I'm not really sure what this comment is saying.